_data/simple-icons.json

```
@@ -12255,8 +12255,8 @@
 	},
 	{
 		"title": "Plausible Analytics",
-		"hex": "5850EC",
-		"source": "https://github.com/plausible/docs/blob/be5c935484e075f1e0caf3c9b3351ddd62348139/static/img/logo.svg"
+		"hex": "1F2961",
+		"source": "https://github.com/plausible/docs/pull/471"
```

Please, use the full URL to the file:

```suggestion
		"source": "https://github.com/plausible/docs/blob/cf53502f7e7a8673f75e3f05517f8a61d5cb7953/static/img/plausible_logo.svg"
```

---

Thank you for your feedback, it should be good now.